.pre-commit-config.yaml

```diff
@@ -7,19 +7,13 @@ default_language_version:
 
 files: |
     (?x)^(
-        .github/|
-        cobalt/|
-        media/base/starboard/|
-        media/starboard/|
-        starboard/|
-        third_party/blink/renderer/modules/cobalt/|
-        third_party/blink/web_tests/wpt_internal/h5vcc/|
```

Comment:
This is still needed right now

Reply:
How so? It doesn't exist after #4980:
https://github.com/youtube/cobalt/tree/1d59c3cc/third_party/blink/web_tests/wpt_internal/h5vcc

Reply:
Ah, didn't realize it got renamed to cobalt